repos/system_upgrade/common/actors/opensshconfigscanner/libraries/readopensshconfig.py

```diff
@@ -68,8 +89,26 @@ def parse_config(config):
                 # here we need to record all remaining items as command and arguments
                 ret.subsystem_sftp = ' '.join(el[2:])
 
+        elif el[0].lower() == 'include':
+            # recursively parse the given file or files referenced by this option
+            # the option can have several space-separated filenames with glob wildcards
+            for pattern in shlex.split(' '.join(el[1:])):
+                if pattern[0] != '/' and pattern[0] != '~':
+                    pattern = os.path.join('/etc/ssh/', pattern)
+                files_matching_include_pattern = glob.glob(pattern)
+                # OpenSSH sorts the files lexicographically
+                files_matching_include_pattern.sort()
+                for included_config_file in files_matching_include_pattern:
+                    output = read_sshd_config(included_config_file)
+                    if parse_config(output, base_config=ret, current_cfg_depth=current_cfg_depth + 1) is None:
+                        raise StopActorExecutionError(
+                            'Failed to parse sshd configuration file: ',
```

Comment:
Just a tiny detail: the message here seems to be intended to also include the filename of the problematic config (guessing by the trailing `: `). Maybe a `.format` is missing?

Reply:
Yeah. Its now in the details so I am not sure if it would be useful to duplicate it also to the error title. What would be the best to be consistent with the rest of the messages?
I think I remember I heard keeping the title static and expanding the filenames in `details` is preferred. If so, please confirm and I will just remove the trailing `: ` from the title.

Reply:
Removed trailing colon.